src/client/Microsoft.Identity.Client/AuthScheme/PoP/PoPAuthenticationScheme.cs

```diff
@@ -49,6 +49,8 @@ public PopAuthenticationScheme(PoPAuthenticationConfiguration popAuthenticationC
             KeyId = Base64UrlHelpers.Encode(keyThumbprint);
         }
 
+        public int Id => 2;
```

Comment:
Should this AuthScheme and the BrokerAuthScheme have the same value? This one is used for CCA and the other one is used for broker.

Reply:
Since this is logging to telemetry to see which auth scheme is used I think this should be fine. @bgavrilMS should this be another value for broker?

Reply:
Discussed offline with Bogdan. We can have the same value. If there a need to differentiate we can use the app id to know if it is public client or confidential client.